operator/go.mod

```diff
@@ -5,72 +5,73 @@ go 1.18
 require (
 	github.com/gardener/component-spec/bindings-go v0.0.65
 	github.com/go-logr/logr v1.2.3
-	github.com/imdario/mergo v0.3.12
+	github.com/imdario/mergo v0.3.13
 	github.com/kyma-project/kyma-watcher/listener v0.0.0-20220714080306-99a0a5162c8b
+	github.com/kyma-project/manifest-operator/operator v0.0.0-20220726131454-83e937e2eb36
```

Comment:
Why is this dependency in? I think you always use string-based GVKs?

Reply:
wait until we have a decision if introduce manifest operator as dependency.